install/kubernetes/cilium/charts/config/templates/configmap.yaml

```diff
@@ -386,6 +386,15 @@ data:
   # https://github.com/cilium/hubble/blob/master/Documentation/metrics.md
   hubble-metrics: {{ .Values.global.hubble.metrics | join " " | quote }}
 {{- end }}
+{{- range .Values.global.hubble.listenAddresses }}
+        - --listen-client-urls={{ . }}
+{{- end }}
+{{- if and .Values.hubble-ui.enabled (not (has ":50051" .Values.global.hubble.listenAddresses)) }}
```

Comment:
I think what you want to do here is to extend `hubble-listen-addresses` (see line 369) here. The if condition itself looks correct.